### PR TITLE
DT-1846 display filter

### PR DESCRIPTION
**Pull request by mikehalmamoj**
Show the filter in the UI - it doesn't do anything yet and doesn't show all court types, coming in next PR.
image

### CODE REVIEW

File: server/routes/courtRegister/courtMapper.ts [hunk 1]
```diff
@@ -85,5 +89,15 @@ export function courtsPageMapper(courtsPage: CourtsPage): CourtsPageView {
     courtsPage.numberOfElements,
     '/court-register/paged?page=:page'
   )
-  return { courts, pageMetaData }
+  const allCourtsFilter: AllCourtsFilter = { courtTypeIds: [] }
+  const courtTypes: CourtType[] = [
```
Comment (mikehalmamoj) [inline server/routes/courtRegister/courtMapper.ts:93]: To be replaced by the court types retrieved form the server

**Review by Mjwillis — APPROVED** [review]
LGTM

File: server/utils/nunjucksSetup.ts [hunk 2]
```diff
@@ -138,52 +141,56 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
...
-      return {
-        href: '#',
-        text: courtTypes.filter(courtType => courtType.courtType === courtTypeId)[0].courtName,
+  njkEnv.addFilter(
+    'toCourtListFilter',
+    (courtTypes: CourtType[], allCourtsFilter: AllCourtsFilter, filterOptionsHtml: string) => {
+      const courtTypeItems = allCourtsFilter.courtTypeIds.map(courtTypeId => {
+        return {
+          href: '#',
+          text: courtTypes.filter(courtType => courtType.courtType === courtTypeId)[0].courtName,
```
Comment (andymarke) [inline server/utils/nunjucksSetup.ts:150]: I think you want ‘find’ rather than ‘filter’

**Review by andymarke — APPROVED** [review]
Lgtm

**Pull request merged**